cmd/manager/main.go

```diff
@@ -75,7 +75,7 @@ func main() {
 	ctx := context.TODO()
 
 	// Become the leader before proceeding
-	err = leader.Become(ctx, "node-maintenance-operator-lock")
```

Comment:
I don't know much about NMO and it's leader election, feel free to silence me ;-)

Still, this namechange smells dangerous: and old copy of NMO may think that its the leader because it holds `node-maintenance-operator-lock`, but a new copy may already be the leader since it holds `openshift-kni-lifecycle-node-maintenance-operator-lock`.

Reply:
you're right, kindly ignore this PR